Source/WebCore/platform/graphics/controls/ControlStyle.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All Rights Reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+ * OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#pragma once
+
+#include "Color.h"
+
+namespace WebCore {
+
+struct ControlStyle {
+    enum class Flag : uint16_t {
+        Hovered         = 1 << 0,
+        Pressed         = 1 << 1,
+        Focused         = 1 << 2,
+        Enabled         = 1 << 3,
+        Checked         = 1 << 4,
+        Default         = 1 << 5,
+        WindowInactive  = 1 << 6,
+        Indeterminate   = 1 << 7,
+        SpinUp          = 1 << 8, // Sub-state for HoverState and PressedState.
+        Presenting      = 1 << 9,
+        SemanticContext = 1 << 10,
```

Comment:
`FormSemanticContext`?

Couple of other places that are `useSemanticContext` should probably be `useFormSemanticContext` too.

Reply:
Fixed. FormSemanticContext and useFormSemanticContext will be used.